apps/lib/apps.c

```diff
@@ -1648,7 +1648,7 @@ X509_NAME *parse_name(const char *cp, int chtype, int canmulti,
             *bp++ = *cp++;
         if (*cp == '\0') {
             BIO_printf(bio_err,
-                       "%s: Hit end of %d name string before finding the '='\n",
+                       "%s: End of %d name string before finding the '='\n",
```

Comment:
`"%s: %d name string doesn't contain a '='\n"`

Reply:
> `"%s: %d name string doesn't contain a '='\n"`

This is not general enough because there may be occurrences of `=` (for other RDNs) before."
Just improved this to:
```
            BIO_printf(bio_err,
                       "%s: Missing '=' after RDN type string '%s' in %s name string\n",
                       opt_getprog(), typestr, desc);

```
(which required terminating the `typestr` just before that).

Reply:
Even better!